spec/requests/internal/broadcasts_spec.rb

```diff
@@ -3,7 +3,7 @@
 
 RSpec.describe "/internal/broadcasts", type: :request do
   let(:get_resource) { get "/internal/broadcasts" }
-  let(:params) { { title: "Hello!", processed_html: "<pHello!</p>", type_of: "Welcome", sent: true } }
+  let(:params) { { title: "Hello!", processed_html: "<pHello!</p>", type_of: "Welcome", active: true } }
```

Comment:
I spy a missing closing `>` here! 👀

Reply:
Ah, this was pre-existing! But I have pushed up the fix.